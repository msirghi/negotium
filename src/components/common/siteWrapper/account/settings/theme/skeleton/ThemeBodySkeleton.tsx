import { Skeleton } from '@mui/material';
import { useThemeItemStyles } from '../item/styles';
import { Row } from '../../../../../utilities/row/Row';

export const ThemeBodySkeleton = () => {
  const classes = useThemeItemStyles({});

  return (
    <Row alignVerticalCenter>
      <Skeleton variant="rectangular" width={10} height={9} animation={false} />
      <Skeleton
        className={classes.textSkeleton}
        width={'100%'}
        height={15}
        animation={false}
      />
    </Row>
  );
};
