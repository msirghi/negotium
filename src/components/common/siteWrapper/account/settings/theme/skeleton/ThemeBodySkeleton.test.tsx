import renderer from 'react-test-renderer';
import { ThemeBodySkeleton } from './ThemeBodySkeleton';

describe('ThemeBodySkeleton', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<ThemeBodySkeleton />);
    expect(tree).toMatchSnapshot();
  });
});
