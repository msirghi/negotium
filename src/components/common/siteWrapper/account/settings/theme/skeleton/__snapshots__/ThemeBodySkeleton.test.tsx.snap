// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeBodySkeleton should match the snapshot 1`] = `
<div
  onClick={[Function]}
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "justifyContent": "flex-start",
      "width": "initial",
    }
  }
>
  <span
    className="MuiSkeleton-root MuiSkeleton-rectangular css-14wtl70-MuiSkeleton-root"
    style={
      Object {
        "height": 9,
        "width": 10,
      }
    }
  />
  <span
    className="MuiSkeleton-root MuiSkeleton-text makeStyles-textSkeleton-4 css-1l6af8c-MuiSkeleton-root"
    style={
      Object {
        "height": 15,
        "width": "100%",
      }
    }
  />
</div>
`;
